src/libmscore/undo.h

```diff
@@ -256,6 +256,22 @@ class RemovePart : public UndoCommand
     UNDO_NAME("RemovePart")
 };
 
+//---------------------------------------------------------
+//   SetSoloist
+//---------------------------------------------------------
+
+class SetSoloist : public UndoCommand
+{
+    Part* part;
```

Comment:
Please, add default values here:

Part* part = nullptr;
bool soloist = false;

Reply:
Good point. However, it seems none of the classes in this file supplied a default value. Might be an idea to go over this file and supply default values for all members of all other classes.

Reply:
Done.